client/app/components/template-explorer/template-explorer.component.js

```diff
@@ -0,0 +1,204 @@
+/* eslint camelcase: "off" */
+export const TemplateExplorerComponent = {
+  controller: ComponentController,
+  controllerAs: 'vm',
+  templateUrl: 'app/components/template-explorer/template-explorer.html',
+};
+
+/** @ngInject */
+function ComponentController($filter, lodash, ListView, Language, Templates, EventNotifications, Session, RBAC, ModalService,
+                             CollectionsApi, sprintf, Polling, $log) {
```

Comment:
Is $log used anywhere? 
The service named "Templates" is concerning, perhaps "TemplatesService" (looking for more specificity)